src/platforms/common/user-feedback/configuration.mdx

```diff
@@ -0,0 +1,260 @@
+| `inputForeground`       | `--input-foreground`        | `inherit`                                 | `inherit`                                 | Foreground color for form inputs                                                                      |
+| `inputBorder`           | `--input-border`            | `var(--border)`                           | `var(--border)`                           | Border styles for form inputs                                                                         |
+| `inputOutlineFocus`     | `--input-outline-focus`     | `rgba(108, 95, 199, 1)`                   | `rgba(108, 95, 199, 1)`                   | Outline color for form inputs when focused                                                            |
+
+Here is an example of customizing only the background color for the light theme using the Feedback constructor configuration.
+
+```javascript
+new Feedback({
+  themeLight: {
+    background: "#cccccc",
+  },
+});
+```
+
+Or the same example above but using the CSS variables method:
+
+```css
+#sentry-feedback {
+  --background: #cccccc;
+}
+```
+
+### Additional UI Customization
+
+Similar to theme customization above, these are additional CSS variables that can be overridden. Note these are not supported in the constructor.
+
+| Variable        | Default                                 | Description                                                                       |
+| --------------- | --------------------------------------- | --------------------------------------------------------------------------------- |
+| `--bottom`      | `1rem`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--right`       | `1rem`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--top`         | `auto`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--left`        | `auto`                                  | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--z-index`     | `100000`                                | The z-index of the widget                                                         |
+| `--font-family` | `"'Helvetica Neue', Arial, sans-serif"` | Default font-family to use                                                        |
+| `--font-size`   | `14px`                                  | Font size                                                                         |
+
+### Event Callbacks
+
+Sometimes it’s important to know when someone has started to interact with the feedback form, so you can add custom logging, or start/stop background timers on the page until the user is done.
+
+Pass these callbacks when you initialize the Feedback integration:
+
+```javascript
+new Feedback({
+  onActorClick: () => {},
```

Comment:
What does this one do? It's the least obvious to me right now

Reply:
I suppose it's also bit redundant, would there be the case when we want a callback when button is clicked and not when dialog is opened?

Reply:
IMO they can make their own button for that, they're already writing js code in that case.

But fine I could be wrong, maybe they want a pre-form-visible thing. Then the name might need help, What about 'onAutoInjectButtonClick' or 'onBeforeFormShow' and what's the return type supposed to be?

Reply:
Err meant 'onBeforeDialogOpen'

Reply:
Yeah I think we can just remove this, seems unlikely you'd need this in addition to `onDialogOpen`